### ISSUE TITLE
Edges should prob not use bounding box intersection to determine selection

### ISSUE THREAD

**Issue by surma** [breadboard-ai/breadboard#3636]
If you have long edges, you can end up selecting edges in a way that is very confusing.
![Screenshot 2024-10-27 at 15 36 16](https://github.com/user-attachments/assets/84bf7389-1124-4ad2-b607-91c54b5464a6)

**Comment by paullewis** [comment]
Yes, unfortunately the click detection in Pixi means that, despite defining the hit area as a Polygon, it seems to use the bounding box of the poly as the click target.

**Comment by paullewis** [comment]
That said I could potentially do manual click tracking as there is a `contains` method.

**Comment by paullewis** [comment]
Then again if it's a box selection I don't know we have that freedom without implementing some fairly extensive box-poly collision code.

**Comment by paullewis** [comment]
So having checked we definitely use the poly for clicks, but the box intersection has to use edge bounds. I'm not thrilled about it, but I don't think we're going to get to building out line/poly-box intersections any time soon so I'm going to close this one.
